semgrep/semgrep/output.py

```diff
@@ -248,13 +248,16 @@ def _final_raise(self, ex: Optional[Exception], error_stats: Optional[str]) -> N
             if ex.level == Level.ERROR:
                 raise ex
             else:
-                if self.settings.strict:
-                    raise ex
+                message = f"{error_stats}; run with --verbose for details"
+                if not self.settings.strict:
+                    message += " or run with --strict to exit non-zero if any file cannot be analyzed cleanly"
                 logger.info(
                     terminal_wrap(
-                        f"{error_stats}; run with --verbose for details or run with --strict to exit non-zero if any file cannot be analyzed cleanly"
+                        message
```

Comment:
I think maybe pre-commit got skipped here?

Reply:
Yeah, sorry. I missed that section of the contributing instructions. @emjin pointed them out in my other PR but I haven't yet circled back to this PR to double-check it. I'll get that sorted out.